### PR TITLE
Added nearObject feature

### PR DESCRIPTION
**Pull request by miniMaddy**
@iamleonie, I have added the nearObject support to the repo. It's present in the nearObject.py file.

### CODE REVIEW

**Comment by iamleonie** [comment]
Hi @miniMaddy, Thanks for another great contribution. This PR is in review. It might take a couple more days. Thank you for your patience.

**Comment by iamleonie** [comment]
Hey @miniMaddy, this is unrelated to the change you made, but to test your change, I ran the import.py file overnight, and it didn't finish. 
Did the import.py file run smoothly for you, and how much time did it take?

**Comment by miniMaddy** [comment]
Hello @iamleonie , the import.py file works smoothly for me. It is done in 15 minutes. I am also able to run near_text queries.

File: requirements.txt [hunk 1]
```diff
@@ -1,26 +1,54 @@
 annotated-types==0.5.0
+asttokens==2.4.0
```
Comment (iamleonie) [inline requirements.txt:2]: Are all of these new additions really necessary? I tried it with the old requirements.txt and it worked nicely.

**Comment by iamleonie** [comment]
Hey @miniMaddy, I finally managed to make it work. It ran smoothly. I have added one remark regarding the requirements.txt. Please have a look.

File: requirements.txt (hunk 1 above)
Comment (miniMaddy) [reply requirements.txt:2]: Hey, I don't think they are necessary, I will just remove them and check once.

**Review by iamleonie — APPROVED** [review]

**Comment by iamleonie** [comment]
Related to https://github.com/weaviate-tutorials/DEMO-text-search-wines/issues/1

**Pull request merged**